Update comm call schedule and add new zoom link for weekly sync. (#274)
dapr/community: README.md

To accomodate different timezones, we schedule community calls at different times zone. See below for the time of each call (note you can always catch up offline by watching the recordings on the Dapr YouTube channel) 

<!--
!!!REMINDER!!!

> DO NOT ADD PASSWORD TO THE MEETING URL TO AVOID ZOOM-BOMBING (yes, we had that before).

Reviewer: ```suggestion
Do not add passcode to the Zoom URL. 
```
 I would not bring up the past. Simply say a reminder.

Use the upcoming calendar events below to add the community call events to your calendar.

> - Wednesday Jan 25<sup>th</sup> 9:00am Pacific Time (PST) - [See it in your local time and add to your calendar](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230125T09&p1=234&msg=Dapr+Community+Call)

Author: We are forgetting to update the URL. See how I had to fix the previously scheduled calls too.

Reviewer: Yes, this was cut and pasted. I will reminder @nyemade-uversky on this.

Every week, we meet to plan and sync on the progress for the milestone's release. This meeting includes maintainers and active contributors and usually lasts for about 30-60 minutes. If you would like to **actively contribute** towards a milestone, you are welcome to join that meeting too.

> **Zoom Call:** https://zoom.us/j/91940016938<br>

Author: The community call meeting requires someone to login as host to start the call. It is too much formality for weekly sync when we not always have someone with access to the host creds.

Reviewer 2: Agree on not requiring a host for weekly sync calls. Too much ceremony

**Meeting ID:** 919 4001 6938<br>

> **Passcode:** 152697<br>

Author: I am keeping the passcode but if we have zoom-bombing incidents again, we will need to generate a new code and keep it hidden and only given on-demand, like before.

Every week, we meet to plan and sync on the progress for the milestone's release. This meeting includes maintainers and active contributors and usually lasts for about 30-60 minutes. If you would like to **actively contribute** towards a milestone, you are welcome to join that meeting too.

<!--
We are using a different link from Community Call because the Community Call requires the host to admit each person. In the weekly syncs, we "never" login with the host account.
-->

> **Zoom Call:** https://zoom.us/j/91940016938<br> <!-- DO NOT ADD PASSWORD TO THE MEETING URL TO AVOID ZOOM-BOMBING (yes, we had that before). -->

Reviewer: ```suggestion
**Zoom Call:** https://zoom.us/j/91940016938<br> <!-- Do not add password to the Zoom URL. -->
```
No need to bring up the past and just have a note.

**Meeting ID:** 919 4001 6938<br>

> **Passcode:** 152697<br> <!-- If we get Zoom-bombing, generate a new one and only share on request, not here>

Reviewer: We have restriction on the call itself to prevent this, so I really would not fixate on the passcode here. The restrictions are better with having a muted callers by default and wait room etc. 
```suggestion
**Passcode:** 152697<br> 
```
